external/glfw/lib/android/android_window.c

```diff
@@ -290,14 +291,17 @@ static void _glfwPlatformSwapBuffersNoLock( void )
             }
         }
     }
-
     /*
-     The preferred way of handling orientation changes is probably
-     in APP_CMD_CONFIG_CHANGED or APP_CMD_WINDOW_RESIZED but occasionally
-     the wrong previous orientation is reported (Tested on Samsung S2 GTI9100 4.1.2).
-     This might very well be a bug..
+     Handle orientation/size changes when signaled by APP_CMD_CONFIG_CHANGED,
+     APP_CMD_WINDOW_RESIZED or APP_CMD_CONTENT_RECT_CHANGED. Some devices
+     report the old EGL size at the time of the event, so we defer the query
+     until a swap occurs.
      */
-    update_width_height_info(&_glfwWin, &_glfwWinAndroid, 0);
+    if (g_PendingResize)
+    {
+        update_width_height_info(&_glfwWin, &_glfwWinAndroid, 1);
+        g_PendingResize = 0;
```

Comment:
it's enough to do it when we know something has changed. We don't have to do it every swap